tutorial/westeros/westeros_baseline.ipynb

```diff
@@ -697,34 +741,21 @@
     }
    },
    "source": [
-    "The model can further be provided `technical_lifetime`s in order to properly manage deployed capacity and related costs via the **capacity maintenance** constraint:\n",
+    "The model can further be provided `\"technical_lifetime\"`s in order to properly manage deployed capacity and related costs via the **capacity maintenance** constraint:\n",
     "\n",
     "$\\text{CAP}_{n,t,y^V,y} \\leq \\text{remaining_capacity}_{n,t,y^V,y} \\cdot \\text{value} \\quad \\forall \\quad t \\in T^{INV}$\n",
     "\n",
     "where `value` can take different forms depending on what time period is considered:\n",
     "\n",
-    "| Value                               | Condition                                           |\n",
-    "|-------------------------------------|-----------------------------------------------------|\n",
-    "| $\\Delta_y \\text{historical_new_capacity}_{n,t,y^V}$ | $y$ is first model period           |\n",
-    "| $\\Delta_y \\text{CAP_NEW}_{n,t,y^V}$ | $y = y^V$                                           |\n",
-    "| $\\text{CAP}_{n,t,y^V,y-1}$          | $0 < y - y^V < \\text{technical_lifetime}_{n,t,y^V}$ |\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "slideshow": {
-     "slide_type": "skip"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "base_technical_lifetime = {\n",
-    "    'node_loc': country,\n",
-    "    'year_vtg': model_horizon,\n",
-    "    'unit': 'y',\n",
-    "}"
+    "| Value                                               | Condition                                           |\n",
+    "|:----------------------------------------------------|:----------------------------------------------------|\n",
```

Comment:
Here I noticed that the table looked strange, i.e. everything was right-aligned in my Jupyter/browser.

I found https://stackoverflow.com/questions/48655801/tables-in-markdown-in-jupyter which suggested using `|:--|` as a column specifier to explicitly force left alignment, whereas `|---|` is implicit and seems to be handled differently by Jupyter than other Markdown implementations.

There are many other tables in the tutorials, and I'm not asking that you revise all of these in order to approve the PR; just making a note for posterity.

Reply:
I just applied it for all the tutorials, so we can put it out of our mind.